src/openms/include/OpenMS/ANALYSIS/ID/PeptideIndexing.h

```diff
@@ -677,8 +699,155 @@ namespace OpenMS
       return EXECUTION_OK;
     }
 
-protected:
-    struct PeptideProteinMatchInformation
+ protected:
+
+     using MapDecoyStringToPrefixSuffixCount = std::map<std::string, std::pair<int, int>>;
```

Comment:
ThatsQuiteAMouthfulAndHardToRead :-)
How about `DecoyCounts`

Reply:
Another contributor suggested this. I personally don't mind, so that's up to you guys ;)

Reply:
I would vote for DecoyStringToAffixCount. The "To" implies a Map and Affix combines Suffix and Prefix.